packages/syntax-core/src/getZIndex.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * [getZIndex](https://cambly-syntax.vercel.app/?path=/docs/ZIndex--docs) is a utility function that returns a z-index value based on the layer and local layer.
```

Comment:
Note that in this case, we do need to leave the URL as is since otherwise it won't work with TSDoc